src/components/AvatarShape.js

```diff
@@ -77,18 +82,20 @@ const ShapeOutline = styled(Link)`
 const AvatarShape = props => (
   <ShapeOutline shape={props.shape} bg={props.bg} {...props}>
     <A className={props.className} shape={props.shape}>
-      <Image src={props.src} />
+      <Image src={props.src} $hover={props.hover} />
```

Comment:
Do you need $ prefix here? ☝️

Reply:
The $ turns it into a [transient prop](https://styled-components.com/docs/api#transient-props) so it's not output as an attribute on the DOM element. I just realised that we're not on styled v5.1 so this won't make a difference right now 😅  but when we do upgrade the package it'll make the html a teeny bit more valid.

Reply:
Nice! TIL 🙌